src/checks/stylelint.ts

```diff
@@ -0,0 +1,65 @@
+import { execaCommand } from 'execa';
+import { LintResult } from 'stylelint';
+import { projectPackageJson } from '../util/packageJson';
+
+const projectDependencies = projectPackageJson.dependencies || {};
+
+export const supportedFileExtensions = [
```

Comment:
since we are exporting / importing this
```suggestion
export const supportedStylelintFileExtensions = [
```